content/kubernetes/deployment/quick-start.md

```diff
@@ -183,26 +176,124 @@ You can test the operator by creating a minimal cluster by following this proced
 
     ```sh
     NAME           AGE
-    test-cluster   1m
+    test-rec   1m
     ```
 
    At this point, the operator will go through the process of creating various
-   services and pod deployments. You can track the progress by examining the
+   services and pod deployments.
+
+   You can track the progress by examining the
    StatefulSet associated with the cluster:
 
    ```sh
-   kubectl rollout status sts/test-cluster
+   kubectl rollout status sts/test-rec
    ```
 
-   or by simply looking at the status of all of the resources in your namespace:
+   or by looking at the status of all of the resources in your namespace:
 
    ```sh
    kubectl get all
    ```
 
-<!---ADD Admission Controller and namespace limiting webhook steps here--->
+<!--- Once the cluster is up, the cluster GUI and API could be used to configure databases. It is recommended to use the K8s REDB API that is configured through the following steps. To configure the cluster using the cluster GUI/API, use the ui service created by the operator and the default credentials as set in a secret. The secret name is the same as the cluster name within the namespace.--->
```

Comment:
Is this supposed to be commented out still?

Reply:
Thanks for catching that. It's extra info that doesn't need to be there.